### ISSUE TITLE
Issues with FA2 when adjustSizes is set to true

### ISSUE THREAD

**Issue by apitts** [jacomyal/sigma.js#720]
I have been trying to get adjustSizes to work. So far, I have found two things which I believe are issues. 

Firstly, the '+' here and on the line below should be minuses I think: 
https://github.com/jacomyal/sigma.js/blob/master/plugins/sigma.layout.forceAtlas2/worker.js#L650

Secondly, I believe the sizes in the distance calculation below should be taken out of the square root: 
https://github.com/jacomyal/sigma.js/blob/master/plugins/sigma.layout.forceAtlas2/worker.js#L748

But...alas, after making these two changes and also after changing x, y and size to use the rendered versions....I still haven't been able to get adjustSizes to work for FA2. Nodes basically stay overlapped and jitter around a little but not much else. The actual application of the computed forces is quite opaque to me...so perhaps there is a problem there...I'm not sure. 

Has anyone got it to work before?  Any thoughts @Yomguithereal on how to get this option working or have you seen it working in sigma.js before? Thanks!

**Comment by apitts** [comment]
OK...I found a third issue and it looks like that has done it :smiley: 

Pleased to report that I now have adjustSizes working. @Yomguithereal - let me know if you would like a pull request for these fixes.

**Comment by Yomguithereal** [comment]
So you're saying there are differences between the Gephi version and sigma's one concerning computations? Can you open a PR so I can check that and fix if necessary please?

**Comment by apitts** [comment]
Yes - those three issues plus I think necessary to prefix x, y and size with the renderer prefix for the purpose of adjustSizes. Haven't quite found an elegant way to do the last part put I will submit a PR with the three issues and a hacky way to do the latter which I'm sure you will have an idea to improve on.  May take a few days to get to it but will do it. Also, it looks like adjustSizes may still not be working with Barnet-Hut active but not certain...will have to look at it again.
